data-api/open-api-specification.yml

```diff
@@ -226,8 +226,22 @@ components:
           $ref: '#/components/schemas/CaseDetails'
         ClientDetails:
           $ref: '#/components/schemas/ClientDetails'
+        CapitalDetails:
+          $ref: '#/components/schemas/CapitalDetails'
```

Comment:
All this schemas should be named with suffix DTO. And would also help having a prefix like Crm5 (or Crm, but I think we should do Crm5 for now since we do not know if they are really fully reusable yet)

Reply:
I would prefer it to be similar to this:
```
CapitalDetails:
           $ref: '#/components/schemas/Crm5CapitalDetailsDTO'
```

even `$ref: '#/components/schemas/Crm5CapitalDTO'` would work

Reply:
We agreed in changing the names including Crm5 prefix and DTO suffix